include/podio/RNTupleWriter.h

```diff
@@ -1,25 +1,38 @@
 #ifndef PODIO_RNTUPLEWRITER_H
 #define PODIO_RNTUPLEWRITER_H
 
+#include "TFile.h"
 #include "podio/CollectionBase.h"
 #include "podio/Frame.h"
 #include "podio/GenericParameters.h"
 #include "podio/SchemaEvolution.h"
 #include "podio/utilities/DatamodelRegistryIOHelpers.h"
 
-#include "TFile.h"
 #include <ROOT/RNTuple.hxx>
 #include <ROOT/RNTupleModel.hxx>
+
 #include <RVersion.h>
+#include <cstdint>
+
+namespace ROOT {
+namespace Experimental {
+  class REntry;
+} // namespace Experimental
+} // namespace ROOT
 #if ROOT_VERSION_CODE >= ROOT_VERSION(6, 31, 0)
   #include <ROOT/RNTupleWriter.hxx>
 #endif
 
+#include <memory>
 #include <string>
+#include <tuple>
 #include <unordered_map>
+#include <utility>
 #include <vector>
 
 namespace podio {
+class Frame;
+class GenericParameters;
```

Comment:
We have the includes above for these and I think these forward declarations are not necessary (or we can remove the includes).